testing/zerotier-one/APKBUILD

```diff
@@ -1,38 +1,48 @@
-# Contributor: Kyle Parisi <kyleparisi@gmail.com>
-# Maintainer: Kyle Parisi <kyleparisi@gmail.com>
+# Contributor: ZeroTier, Inc. <contact@zerotier.com>
+# Maintainer: Adam Ierymenko <adam.ierymenko@zerotier.com>
+
+## Thanks to Kyle Parisi <kyleparisi@gmail.com> for doing the original package and getting it into testing
+
 pkgname=zerotier-one
-pkgver=1.1.14
-pkgrel=1
-pkgdesc="ZeroTier One allows systems to join and participate in ZeroTier virtual networks."
+pkgver=1.2.4
+pkgrel=0
+pkgdesc="Join ZeroTier virtual neteworks from your Alpine Linux system"
 url="https://www.zerotier.com/"
 arch="all"
 license="GPL3"
-depends=""
-makedepends="nodejs linux-headers"
-install=""
-subpackages="$pkgname-doc"
+depends="libstdc++"
+makedepends="linux-headers"
+install="$pkgname.pre-install $pkgname.post-deinstall"
+subpackages=""
 source="
-	$pkgname-$pkgver.tar.gz::https://github.com/zerotier/ZeroTierOne/archive/${pkgver}.tar.gz
 	zerotier-one.initd
+	zerotier-one.confd
+	${pkgname}-${pkgver}.tar.gz::https://github.com/zerotier/ZeroTierOne/archive/${pkgver}.tar.gz
 	"
+builddir="$srcdir/ZeroTierOne-${pkgver}"
 
-builddir="$srcdir"/ZeroTierOne-$pkgver
 build() {
 	cd "$builddir"
-	sed -i.bak s:/bash:/sh:g ./doc/build.sh
-	make || return 1
+	make distclean
+	make -j4 one
```

Comment:
abuild automatically sets the JOBS variable, so specifying `-j` manually is both unnecessary and potentially slowing down (the ppc64le build server has far more than 4 cores, for instance :).